less/components/header.less

```diff
@@ -1,4 +1,15 @@
-.header {
+.page-header {
+    padding: 0 1em;
+    color: #FFFFFF;
```

Comment:
Because it doesn't *mean* anything as a variable. We're not going to change the value of `@white` to be anything other than `#FFF`. So all this does is add one more little bit of code (defining the variable).